TransIT.DAL/Models/DTOs/UserDTO.cs

```diff
@@ -2,12 +2,11 @@
 {
     public class UserDTO
     {
-        public int Id { get; set; }
         public string FirstName { get; set; }
         public string LastName { get; set; }
         public string Email { get; set; }
         public string PhoneNumber { get; set; }
         public string Login { get; set; }
-        public RoleDTO Role { get; set; }
+        public string Role { get; set; }
```

Comment:
Okay, how will admin modify or remove user with id = 5?

Reply:
Our logins are unique, by them you can find a user, I think we do not need this id on client, do we?

Reply:
It's more **restful** to have an id in the url
Moreover, how will you order users by the creation date? We can use ID for this purpose